api-server/src/server/middlewares/request-authorization.js

```diff
@@ -26,6 +26,7 @@ const updateHooksRE = /^\/hooks\/update-paypal$/;
 // note: this would be replaced by webhooks later
 const donateRE = /^\/donate\/charge-stripe$/;
 const submitCoderoadChallengeRE = /^\/coderoad-challenge-completed$/;
+const mobileLoginRE = /^\/mobile-login/;
```

Comment:
```suggestion
const mobileLoginRE = /^\/mobile-login\/?$/;
```
I think we can be relaxed about the trailing slash, but should be stricter otherwise.